### PR TITLE
Solution to  issue #1743 

### PR DESCRIPTION
**Pull request by zulrich91**
Here is a pull request for issue #1743 Adding an intro to mashup.jsp

### CODE REVIEW

**Comment by kanghj** [comment]
Do have a look at our [development process](https://github.com/TEAMMATES/repo/blob/master/devdocs/process.md). 

The name of the pull request, branch name and description of the pull request is incorrect. Remove the changes that are not related to this issue from the pull request.

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -30,18 +30,20 @@
...
-                        </form>
+                          <form action="/login" name="instructorLogin" style="float: left;">
+                                <input type="submit" name="instructor" class="button" id="btnInstructorLogin" value=" Instructor Login"/>
+                          </form>
+                          
+                          <form action="/login" style="float: left;" name="studentLogin">
+                            <input type="submit" name="student" class="button" id="btnStudentLogin" value="Student Login"/>
+                         </form>
+                        
                    </div>
```
Comment (kanghj) [inline src/main/webapp/index.html]: You should make the changes for the issue in a new branch, instead of the 'tinker' branch

**Comment by zulrich91** [comment]
Sorry for the disagreements , i did not go through our development process guide . 
But, please i will go through development guide and get back to the issue

**Comment by zulrich91** [comment]
Hi Kang

I went through the development process guide and did back the pulll request
!!!
thanks,
Ulrich



On Mon, Mar 16, 2015 at 4:48 PM, Kang Hong Jin <notifications@github.com>
wrote:

> Closed #3142 <https://github.com/TEAMMATES/repo/pull/3142>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TEAMMATES/repo/pull/3142#event-255683735>.
>



-- 
ulrich tchuenkam

**Comment by zulrich91** [comment]
Please have a look at it and give me the feedback


On Thu, Mar 19, 2015 at 2:23 PM, tchuenkam ulrich <ulrichmobi2@gmail.com>
wrote:

> Hi Kang
>
> I went through the development process guide and did back the pulll
> request !!!
> thanks,
> Ulrich
>
>
>
> On Mon, Mar 16, 2015 at 4:48 PM, Kang Hong Jin <notifications@github.com>
> wrote:
>
>> Closed #3142 <https://github.com/TEAMMATES/repo/pull/3142>.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/TEAMMATES/repo/pull/3142#event-255683735>.
>>
>
>
>
> --
> ulrich tchuenkam
>



-- 
ulrich tchuenkam

**Comment by kanghj** [comment]
@zulrich91, I replied to you on the new pull request and left some comments. Take a look there?

**Pull request closed without merging**